batchupload/make_info.py

```diff
@@ -186,7 +186,7 @@ def make_info(self):
                 'cats': cats, 'meta_cats': meta_cats}
         return out_data
 
-    def run(self, in_file, base_name=None):
+    def run(self, in_file, base_name, update_mappings):
```

Comment:
Does "update_mappings" have the same function as "update" above? This might be easier to notice if they're called the same.